### ISSUE TITLE
Error: Could not install template alexa

### ISSUE THREAD

**Issue by aggierunner** [stdlib/lib#48]
When attempting to create a service based on the alexa template, I'm getting an error message.

- OS = Win 10 Pro
- Node = 6.9.5

Error:
```
lib create -t alexa --write-over

Awesome! Let's create a stdlib service!
We'll use the template alexa to proceed.

? Service Name alexa
Fetching template alexa...

Error: Could not install template alexa
```

**Comment by keithwho** [comment]
Hey @aggierunner, thanks for this. I think it's definitely a Win10 issue, will get on it ASAP.

**Comment by aggierunner** [comment]
Thanks, no rush as I got it to work from the Win 10 bash shell for now.

**Comment by Alluuu** [comment]
I get the same, only on Windows 8 and node.js v6.9.5 LTS

**Comment by nhajiyani** [comment]
I am getting the same on Win7 Enterprise and for nodejs 6.9.5 LTS. Attaching a screenshot.
[http://imgur.com/a/wrnTs](http://imgur.com/a/wrnTs)
@aggierunner How did you get it to work for you?

**Comment by keithwhor** [comment]
Hey @Alluuu and @nhajiyani -- the issue is the default Command Line for Windows. Windows Bash should work (http://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/) because we're using a linux-based tar unpack command.

I'll fix this to work natively but it might take a day to get to.

**Comment by nhajiyani** [comment]
@keithwhor Thanks for the quick response bud.
Since I am still on Win 7 , the Win 10 bash is not an option for me. So I will just wait for this patch.

**Comment by keithwhor** [comment]
@aggierunner @Alluuu and @nhajiyani --- this should be fixed in `2.0.6` https://github.com/stdlib/lib/commit/1f6aa617ddc3345c205d14ccdaa97ab043847c5b

Re-install `lib.cli` globally and let me know if you still run into this issue? Will close for now.

**Comment by Alluuu** [comment]
@keithwho Great job, and very quick to jump in and solve!

Can confirm that it works on windows too now, for the time being I just used an old mac I had laying around.
